snooty/postprocess.py

```diff
@@ -1141,6 +1141,39 @@ def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
             return
 
 
+class InstruqtHandler(Handler):
+    """Identify if Instruqt directive is present on a page and add title as a page-level option if so"""
+
+    def __init__(self, context: Context) -> None:
+        super().__init__(context)
+        self.has_instruqt_directive = False
```

Comment:
What behavior do we want if a writer puts two instruqt directives on a page?

Reply:
Per @rayangler's advice, a diagnostic error will be produced